lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php

```diff
@@ -115,7 +114,7 @@ protected function generateMigration(Configuration $configuration, InputInterfac
         ];
         $replacements = [
             $configuration->getMigrationsNamespace(),
-            $version,
+            $version = $configuration->generateVersionNumber(),
```

Comment:
@chrisguitarguy That's the only part of the PR that bothers me. I would prefer that the generateMigration function still gets the version "name" passed to it rather than hardcode it in the method.